### ISSUE TITLE
Update to Font Awesome 5.8

### ISSUE THREAD

**Issue by MichaelMarner** [brianegan/font_awesome_flutter#42]
Any plans to update this to Font Awesome 5.8? It includes a bunch of new icons:

https://fontawesome.com/changelog/latest

**Comment by michaelspiss** [comment]
If you need the new icons _right now_, you can do the same steps I outlined for pro support with the "normal" icons. While the pull request (#37) has not been merged, use the fork: https://github.com/michaelspiss/font_awesome_flutter. But I also hope this package gets updated soon.

**Comment by ghenry** [comment]
5.9 now :)

**Comment by bartektartanus** [comment]
Fixed this in my repo, current workaround:
```yaml
dependencies:
  font_awesome_flutter:
    git:
      url: git@github.com:bartektartanus/font_awesome_flutter.git
```

**Comment by brianegan** [comment]
Fixed by #48
